lib/matplotlib/ticker.py

```diff
@@ -2502,64 +2667,118 @@ def view_limits(self, vmin, vmax):
         return result
 
 
-class LogitLocator(Locator):
+class LogitLocator(MaxNLocator):
     """
     Determine the tick locations for logit axes
     """
 
-    def __init__(self, minor=False):
-        """Place ticks on the logit locations."""
-        self.minor = minor
+    def __init__(self, minor=False, *, nbins="auto"):
+        """
+        Place ticks on the logit locations
+
+        Parameters
+        ----------
+        nbins : int or 'auto', optional
+            Number of ticks. Only used if minor is False.
+        minor : bool, default: False
+            Indicate if this locator is for minor ticks or not.
+        """
+
+        self._minor = minor
+        MaxNLocator.__init__(self, nbins=nbins, steps=[1, 2, 5, 10])
 
-    def set_params(self, minor=None):
+    def set_params(self, *, minor=None, **kwargs):
```

Comment:
minor can't become kwonly, that breaks backcompat (though see cbook._make_keyword_only)

Reply:
Done. Sorry for that, I missed `minor` was previously present.